.pre-commit-config.yaml

```diff
@@ -4,3 +4,8 @@ repos:
       hooks:
           - id: flake8
             additional_dependencies: [flake8-bugbear]
+    - repo: https://github.com/PyCQA/bandit
+      rev: 1.6.2
+      hooks:
+        - id: bandit
+          entry: bandit -ll --exclude=tests/ --skips=B322
```

Comment:
This should be `--skip` not `--skips`.

Reply:
good catch.  Thought I'd gotten this one. 

https://github.com/mitre/caldera/pull/1539/commits/a54214a5698767a32695a3208698db6afcd66d80